src/System.Text.Primitives/System/Text/Parsing/PrimitiveParser.cs

```diff
@@ -43,5 +44,11 @@ public static partial class PrimitiveParser
             0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF,             // 239
             0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF              // 255
         };
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static bool IsHexFormat(ParsedFormat format)
```

Comment:
Because IsHex is not a generic format property, but rather something specific to integers. We want the ParsedFormat struct to be "generic".